pygls/types.py

```diff
@@ -687,6 +687,18 @@ def __init__(self, uri: str, range: 'Range'):
         self.range = range
 
 
+class LocationLink:
```

Comment:
Should this and other changes made in order to support the current LSP specification be mentioned in the changelog?